campos_clientes_vittal/models/singe_datetime.py

```diff
@@ -0,0 +1,204 @@
+
+#print("******************* 02 *******************")
+
+# for name, value in sorted(root.items()):
+#	print('%s = %r' % (name, value))
+
+#print("******************* 03 *******************")
+# print(root[1].keys())
+
+
+
+# for idt,element in enumerate(root.iter()):
+# 	print("%s .- %s || %s" % (idt, element.tag, element.text))
+	# print('**************** item begin {} *****************'.format(idt))
+	# for idx, item in enumerate(element.iter()):
+	# 	print('**************** items begin {} *****************'.format(idx))
+	# 	print("%s .- %s |-| %s" % (idx, item.tag, item.text))
+	# 	print('**************** items end {} *****************'.format(idx))
+	# print('**************** item end {} *****************'.format(idt))
+
+
+#print("******************* 04 *******************")
+# print("**************************************")
+# #print(root.iterfind(".//Transaccion").tag)
+
+# print("**************************************")
+# tree = etree.ElementTree(root)
+# a = root[1]
+# print(tree.getelementpath(a[0]))
+
+
+#root = ET.fromstring(contents_xml)
+# print("tag: {}".format(root.tag))
+# print("attrib: {}".format(root.attrib))
+# #topic=root.find(".//*[@TFD='UUID']").text
+# topic=root.find(".//*[@UUID]").text
+# print(topic)
+
+# print("******************* 05 *******************")
+
+# # for child in root:
+# # 	print child.tag, child.attrib
+
+# print("******************* 06 *******************")
+# for neighbor in root.iter('CFD'):
+# 	print neighbor.attrib
+# print("******************* 07 *******************")
+
+# for country in root.findall('./Body/emitirCFDResponse/emitirCFDResult/ResponseAdmon/Transaccion'):
+# 	rank = country.find('UUID').text
+# 	name = country.get('FechaTimbrado')
+# 	print name, rank
+
+# print("******************* 08 *******************")
+
+# for country in root.findall('{http://namespace.pegasotecnologia.com/SCFD}emitirCFDResult'):
+# 	rank = country.find('{http://www.pegasotecnologia.com/}Transaccion').text
+# 	#name = country.get('FechaTimbrado')
+# 	#print name, rank
+# 	print rank
+
+# print("******************* 09 *******************")
+#tree = ET.fromstring(contents_xml)
+# tree = ET.parse('file_data')
+# root = tree.getroot()
+
+
+# topic=root.find(".//*[@TFD='UUID']").text
+# print(topic)
+
+#print([elem.tag for elem in root.iter()])
+# print("*******************************************")
+# for idx, movie in enumerate(root.iter('{http://www.pegasotecnologia.com/}TFD')):
+# 	print(idx)
+# 	print(movie.attrib)
+# 	tfd=movie.attrib
+
+# #tfd=root.iter('{http://www.pegasotecnologia.com/}TFD')
+# print(tfd['UUID'])
+
+
+# print("*******************************************")
+# for idx, movie in enumerate(root.iter('{http://www.pegasotecnologia.com/}Transaccion')):
+# 	print(idx)
+# 	print(movie.attrib)
+# 	tfd=movie.attrib
+
+# #tfd=root.iter('{http://www.pegasotecnologia.com/}TFD')
+# print(tfd['estatus'])
+
+
+
+#0 .- {http://schemas.xmlsoap.org/soap/envelope/}Envelope || None
```

Comment:
Estas ligas si son importantes se pueden quedar

Reply:
Pero dando un contexto en un comment por ejemplo
`This urls are here in order to ...`